api/envoy/service/cluster/v2alpha/aggregated_event_service.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+
+package envoy.service.cluster.v2alpha;
+
+import "envoy/api/v2/core/base.proto";
+
+import "google/protobuf/any.proto";
+
+import "udpa/annotations/migrate.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.service.cluster.v2alpha";
+option java_outer_classname = "AggregatedEventServiceProto";
+option java_multiple_files = true;
+option java_generic_services = true;
+option (udpa.annotations.file_migrate).move_to_package = "envoy.service.cluster.v3";
+
+// [#protodoc-title: gRPC Aggregated Event Service]
+
+// [#not-implemented-hide:]
+// Service for streaming different types of events from Envoy to a server. The examples of
+// such events may be health check or outlier detection events.
+service AggregatedEventService {
+  // Envoy will connect and send StreamAggregatedEventsMessage messages forever. It does not
+  // expect any response to be sent as nothing would be done in the case of failure. The server
+  // should disconnect if it expects Envoy to reconnect. This API is designed for high throughput
+  // with the expectation that it might be lossy.
+  rpc StreamAggregatedEvents(stream StreamAggregatedEventsMessage)
+      returns (StreamAggregatedEventsResponse) {
+  }
+}
+
+// [#not-implemented-hide:]
+// Empty response for the StreamAggregatedEvents API. Will never be sent. See below.
```

Comment:
Have you considered now or in the future adopting a pattern similar to LRS/HDS, where the event sink might want to negotiate the events of interest after the client (Envoy) advertises them in a request? If this makes sense, I'd make this fully bidi as a future proofing, even if we don't implement right now.

Reply:
+1 for this. I think it would be useful to have the event sink decide that events it is interested in

Reply:
done